server/backend/courses/models.py

```
@@ -36,18 +36,34 @@ class CourseContents(models.Model):
+    order = models.PositiveIntegerField(default=0)
     url = models.URLField(null=True, blank=True)
     text_content = models.TextField(blank=True)
+    # file = models.FileField(upload_to='course_files/', null=True, blank=True)
 
+    class Meta:
+        ordering = ['order']
 
     def __str__(self):
         return f"{self.title}/{self.content_type}"
+
+    def clean(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean`

Add return type annotation: `None`

(ANN201)